Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -2490,6 +2497,7 @@ confirmPickUpLocationView push state =
 
               , currentLocationView push state
               , nearByPickUpPointsView state push
+              , nearByDropPointsView state push
```

Comment:
can we use a single component for this which would be generic for both pick up and drop points